StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsGenericInfoScreen.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  FinancialConnectionsGenericInfoScreen.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 6/20/24.
+//
+
+import Foundation
+
+struct FinancialConnectionsGenericInfoScreen: Decodable {
+
+    let id: String
+    let header: Header?
+    let body: Body?
+    let footer: Footer?
+    let options: Options?
+
+    struct Header: Decodable {
+        let title: String?
+        let subtitle: String?
+        let icon: FinancialConnectionsImage?
+        let alignment: Alignment?
+
+        enum Alignment: String, Decodable {
+            case left = "left"
+            case center = "center"
+            case right = "right"
+        }
+    }
+
+    struct Body: Decodable {
+
+        let entries: [BodyEntry]
+
+        enum BodyEntry: Decodable {
+            case text(TextBodyEntry)
+            case image(ImageBodyEntry)
+            case unparasable
+
+            public init(from decoder: Decoder) throws {
+                let type = try? decoder
+                    .container(keyedBy: TypeDecodingContainer.CodingKeys.self)
+                    .decode(TypeDecodingContainer.self, forKey: .type)
+                    .type
+                let container = try decoder.singleValueContainer()
+                if type == .text, let value = try? container.decode(TextBodyEntry.self) {
+                    self = .text(value)
+                } else {
+                    self = .unparasable
+                }
```

Comment:
Should there also be something along the lines of `else if type == .image { self = .image(...) }` here? Or is that not supported yet?

Reply:
Great catch! Although this part is not used/tested yet because it doesn't get used for this PR, it was straight-up-missing without me noting that

https://github.com/stripe/stripe-ios/pull/3745/commits/f814a35bd3ceaf559b2141fdb285e7da3f0f9073